exe/kubernetes-deploy

```diff
@@ -38,6 +37,7 @@ ARGV.options do |opts|
     exit
   end
   opts.parse!
+  bindings = bindings.parse
```

Comment:
This should be `parser.parse`, which reminds me that someone needs to tophat all affected executables before we merge this.

Reply:
Yeah, I did my tophat with `render` 🤦‍♂️